### DISCUSSION TITLE
Problem with baseline LS estimator in Neural_Receiver.ipynb

### DISCUSSION THREAD

**Discussion by mhni-ES** [NVlabs/sionna#443] [General]
Hi Team Sionna 

I am having issues with your NeuralReceiver.ipynb e2e baseline when I try to simulate 4K QAM at high SNR conditions 25 dB+
I am only changing the carrier frequency and the num_bits_per_symbol in the script. 
I have simulated different carrier frequencies. Here the BLER seems inconsistent for what is expected. 
CSI is always fine but LS estimator seems to have strange behavior when SNR goes above 25 dB and I see that the BLER trends towards BLER = 10e0 from 10e-3 from SNR 25 and beyond. See picture. 
This behavior is not present when carrier freqeuncy is below 3.2 but starts when frequency is equal or above 3.3. 
This happens on A100 both with and without XLA using the sionna.config.xla_compat=False initially in the script to stop xla
And I have also tested on CPU and this behavior is also present there. I can also replicate it on google colab. 

I was expecting the BER and BLER to stay at a floor but not rise like the following shows for carrier_frequency= 3.3e9 
![BLER_issue](https://github.com/NVlabs/sionna/assets/67476685/a7dc6abc-7e26-436d-b44e-023b2a5c0f65)

This behavior is not present when carrier_frequency is below 3.2e9

**Comment by rwiesmayr** [comment]
Hi @mhni-ES,

I understand you've changed the carrier frequency and modulation order to 4096-QAM. I can confirm your observation at 3.5GHz, likely due to the following reasons:

1. **CSI Errors**: With high constellation orders like 4096-QAM, CSI estimation errors become more significant.
2. **Doppler Effect**: The tutorial models mobility (10m/s), causing a dominant Doppler effect at higher frequencies, which is not fully compensated by channel estimation and interpolation at higher SNR.
3. **Channel Estimation**: LMMSE equalization uses a channel estimation error variance that doesn't match the "true" variance due to mobility/Doppler.
4. **LLR Mismatch**: LLRs might be computed on a mismatched model (see above points), affecting probability estimates taken in by the decoder.

You can mitigate this by either turning off mobility (speed=0.0) or choosing different subcarrier spacings.

Hope this helps!

**Comment by SebastianCa** [comment]
This seems to be more of a discussion than an issue with the Sionna code. I’ve converted it to a discussion for further conversation.
